### PR TITLE
Automate merge to develop

### PR DESCRIPTION
**Pull request by barmac**
This will create a PR each time a commit is merged to master. Camunda/modeling-dev is assigned to review. Related to https://github.com/bpmn-io/fake-join/issues/4

### CODE REVIEW

**Comment by barmac** [comment]
This is how it looks in action: https://github.com/barmac/test/pull/1

**Comment by barmac** [comment]
Ready for review.

File: .github/workflows/AFTER_MERGE.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: AFTER_MERGE
```
Comment (nikku) [inline .github/workflows/AFTER_MERGE.yml:1]: Maybe give that a more speaking name, i.e. `MERGE_MASTER_TO_DEVELOP`?

File: .github/workflows/AFTER_MERGE.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Create pull request to develop
+      uses: repo-sync/pull-request@65194d8015be7624d231796ddee1cd52a5023cb3
+      with:
+        source_branch: ""                                   # If blank, default: triggered branch
+        destination_branch: "develop"                       # If blank, default: master
+        pr_title: "Merge master to develop"                 # Title of pull request
+        pr_body: "This pulls ${{ github.ref }} to develop." # Full markdown support, requires pr_title to be set
```
Comment (nikku) [inline .github/workflows/AFTER_MERGE.yml:22]: Just because we're on the web we could actually link here :see_no_evil:.

Comment (barmac) [reply .github/workflows/AFTER_MERGE.yml:22]: What would you expect?

Comment (barmac) [reply .github/workflows/AFTER_MERGE.yml]: I'd love to add a pull request here. However, we react to commit being added. As a result, I cannot give a link to a PR. Also note that not all commits on main/master come from PRs.

Comment (nikku) [reply .github/workflows/AFTER_MERGE.yml]: I'd expect simple linking to happen

```
This pulls [`${{ github.ref }}`](https://github.com/barmac/test/tree/${{ github.ref }}`) into [`develop`](https://github.com/barmac/test/develop).
```

Comment (nikku) [reply .github/workflows/AFTER_MERGE.yml]: Alternatively we could simplify the message:

* "This integrates latest changes from master."

Comment (barmac) [reply .github/workflows/AFTER_MERGE.yml]: Let's use the simplified message. Thanks for the suggestion.

Comment (nikku) [reply .github/workflows/AFTER_MERGE.yml]: One issue is that we do not currently allow merge as a strategy in our PRs. We'd need to adjust that accordingly.

Also we need to make sure to _only allow merge_ as everything else will screw up our history (i.e. prevent us from merging `develop` into `master` without actual commit duplication).

Comment (nikku) [reply .github/workflows/AFTER_MERGE.yml]: As I think about it I'd rather love our [https://github.com/bpmn-io-bot](https://github.com/bpmn-io-bot) to automatically merge and if that fails shoot us a message via slack.

Comment (barmac) [reply .github/workflows/AFTER_MERGE.yml]: Cool idea. I'll try that out.

File: .github/workflows/AFTER_MERGE.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: MERGE_MASTER_TO_DEVELOP
```
Comment (nikku) [inline .github/workflows/AFTER_MERGE.yml:1]: Let's get rid of this one.

**Comment by nikku** [comment]
Will wait for you to try out https://github.com/camunda/camunda-modeler/pull/2344#discussion_r663904423 before approving.

**Comment by barmac** [comment]
@nikku I've implemented an alternative solution. We can close this PR if https://github.com/camunda/camunda-modeler/pull/2349 gets merged.

**Comment by nikku** [comment]
Closing this one in favor of https://github.com/camunda/camunda-modeler/pull/2349.

**Pull request closed without merging**